games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,130 @@
+SUMMARY="Turn-based tactical strategy game with a high fantasy theme"
+DESCRIPTION="Build up a great army, gradually turning raw recruits into \
+hardened veterans. In later games, recall your toughest warriors and form a \
+deadly host whom none can stand against! Choose units from a large pool of \
+specialists, and hand-pick a force with the right strengths to fight well on \
+different terrains against all manner of opposition.
+
+Wesnoth has many different sagas waiting to be played. Fight to regain the \
+throne of Wesnoth, of which you are the legitimate heir... step into the \
+boots of a young officer sent to guard a not-so-sleepy frontier outpost... \
+lead a brutal quest to unite the orcish tribes... vanquish a horde of undead \
+warriors unleashed by a foul necromancer, who also happens to have taken your \
+brother hostage... guide a band of elvish survivors in an epic quest to find \
+a new home... get revenge on the orcs by using the foul art of necromancy...
+
+There are at least two hundred unit types, sixteen races, six major factions, \
+and hundreds of years of history. The world of Wesnoth is absolutely huge and \
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="wesnoth.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix
+	cmd:wesnoth
+	cmd:wesnothd
```

Comment:
You need to set `-DCMAKE_INSTALL_BINDIR=$prefix/bin` for this, otherwise the binaries will end up in `$prefix/bin/x86` and need `$secondaryArchSuffix `here.

Reply:
Need a check on 32bit (and see how this is used hunting down "destination" in the source, if it's using GNUInstallDirs these end up in $prefix/bin/x86 on 32bit.

Reply:
Can't check on 32 bit here, @kenmays has been testing that version so I guess he would know how the build directory is compiled in 32 bit.

Reply:
Weird, before it even starts I get an error on 32bit for gcc_syslibs${secondaryArchSuffix}_devel ...

Reply:
Please let me know how it goes and I'll try to fix the recipe accordingly

Reply:
> Weird, before it even starts I get an error on 32bit for gcc_syslibs${secondaryArchSuffix}_devel ...

It is called `gcc${secondaryArchSuffix}_syslibs_devel.